Taarafo.Core/Brokers/Storages/IStorageBroker.Profiles.cs

```diff
@@ -13,6 +14,7 @@ public partial interface IStorageBroker
     {
         ValueTask<Profile> InsertProfileAsync(Profile profile);
         IQueryable<Profile> SelectAllProfiles();
+        ValueTask<Profile> SelectProfileByIdAsync(Guid profileId);
```

Comment:
@rcruzfreelance I think ideally adding brokers should be in separate PR to keep commit history pure. However, I don't know how critical is this.

Reply:
I agree

Reply:
Hello @ElbekDeveloper thanks for the comment I created a seperate pull request for the Broker https://github.com/hassanhabib/Taarafo.Core/pull/163